src/ontology/edits/oeo-physical.omn

```diff
@@ -9556,6 +9560,59 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1432",
         OEO_00000387
     
     
+Class: OEO_00310025
+
+    Annotations: 
+        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Combined cycle electricity generation is a fuel-powered electricity generation process in which a gas turbine process is followed by an steam-electric process.",
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "A combined cycle electricity generation process is a fuel-powered electricity generation process in which a gas turbine process is followed by an steam-electric process.",
```
For consistency ??
Most processes have "process" in their label, some don't. Can we harmonize that, at least for energy transformation processes?

Reply:
That is fine. But then I'll add the one without "process" as alternative term.

Reply:
This issue is already quite big, so I created a new issue #1455.